br/pkg/restore/split.go

```diff
@@ -409,7 +338,7 @@ func (helper *LogSplitHelper) splitRegionByPoints(
 	}
 
 	helper.pool.ApplyOnErrorGroup(helper.eg, func() error {
-		newRegions, errSplit := regionSplitter.splitAndScatterRegions(ctx, region, splitPoints, false)
+		newRegions, errSplit := regionSplitter.splitAndScatterRegions(ctx, region, splitPoints)
```

Comment:
Now it can directly call `regionSplitter.client.SplitWaitAndScatter` and the function `splitAndScatterRegions` can be removed.